apis/stardust/package.json

```diff
@@ -31,7 +31,7 @@
   ],
   "scripts": {
     "build": "cross-env NODE_ENV=production rollup --config ../../rollup.config.js",
-    "build:dev": "rollup --config ../../rollup.config.js",
+    "build:dev": "(rollup --config ../../rollup.config.js && say 'done') || say 'error compiling'",
```

Comment:
don't add any mac-os only commands (like `say`) it will break in other environments